OpenCV 4.4 -  error: call to implicitly-deleted default constructor of 'cv::gimpl::Op'

##### System information (version)
- OpenCV => 4.4
- Operating System / Platform => Linux OpenMandriva Cooker
- Compiler => LLVM/Clang 10.0.1

##### Detailed description

Trying build new OpenCV 4.4 for OpenMabdriva Cooker and at build time see error: 
`/usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/stl_construct.h:109:38: error: call to implicitly-deleted default constructor of 'cv::gimpl::Op'
`
Full build log here: https://file-store.openmandriva.org/api/v1/file_stores/a96e50f00d1eb39a0ab11bd9da473347a3446f11.log?show=true
##### Steps to reproduce
Install OpenMandriva Cooker
Try build OpenCV 4.4 with Clang 10.0.1

<!-- to add code example fence it with triple backticks and optional file extension
    ```.cpp
    // C++ code example
    ```
 or attach as .txt or .zip file
-->

##### Issue submission checklist

 - [x] I report the issue, it's not a question
   <!--
   OpenCV team works with answers.opencv.org, Stack Overflow and other communities
   to discuss problems. Tickets with question without real issue statement will be
   closed.
   -->
 - [x] I checked the problem with documentation, FAQ, open issues,
       answers.opencv.org, Stack Overflow, etc and have not found solution
   <!--
   Places to check:
   * OpenCV documentation: https://docs.opencv.org
   * FAQ page: https://github.com/opencv/opencv/wiki/FAQ
   * OpenCV forum: https://answers.opencv.org
   * OpenCV issue tracker: https://github.com/opencv/opencv/issues?q=is%3Aissue
   * Stack Overflow branch: https://stackoverflow.com/questions/tagged/opencv
   -->
 - [x] I updated to latest OpenCV version and the issue is still there
   <!--
   master branch for OpenCV 4.x and 3.4 branch for OpenCV 3.x releases.
   OpenCV team supports only latest release for each branch.
   The ticket is closed, if the problem is not reproduced with modern version.
   -->
 - [ ] There is reproducer code and related data files: videos, images, onnx, etc
   <!--
   The best reproducer -- test case for OpenCV that we can add to the library.
   Recommendations for media files and binary files:
   * Try to reproduce the issue with images and videos in opencv_extra repository
     to reduce attachment size
   * Use PNG for images, if you report some CV related bug, but not image reader
     issue
   * Attach the image as archite to the ticket, if you report some reader issue.
     Image hosting services compress images and it breaks the repro code.
   * Provide ONNX file for some public model or ONNX file with with random weights,
     if you report ONNX parsing or handling issue. Architecture details diagram
     from netron tool can be very useful too. See https://lutzroeder.github.io/netron/
   -->

The error from clang 11 is a bit more readable
```
In file included from ../modules/gapi/src/backends/common/serialization.cpp:7:
In file included from /usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/set:60:
In file included from /usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/stl_tree.h:67:
In file included from /usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/ext/alloc_traits.h:34:
In file included from /usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/alloc_traits.h:33:
/usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/stl_construct.h:109:38: error: call to implicitly-deleted default constructor of 'cv::gimpl::Op'
    { ::new(static_cast<void*>(__p)) _Tp(std::forward<_Args>(__args)...); }
                                     ^
/usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/stl_uninitialized.h:567:8: note: in instantiation of function template specialization 'std::_Construct<cv::gimpl::Op>' requested here
                std::_Construct(std::__addressof(*__cur));
                     ^
/usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/stl_uninitialized.h:623:2: note: in instantiation of function template specialization 'std::__uninitialized_default_n_1<false>::__uninit_default_n<cv::gimpl::Op *, unsigned long>' requested here
        __uninit_default_n(__first, __n);
        ^
/usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/stl_uninitialized.h:685:19: note: in instantiation of function template specialization 'std::__uninitialized_default_n<cv::gimpl::Op *, unsigned long>' requested here
    { return std::__uninitialized_default_n(__first, __n); }
                  ^
/usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/vector.tcc:627:8: note: in instantiation of function template specialization 'std::__uninitialized_default_n_a<cv::gimpl::Op *, unsigned long, cv::gimpl::Op>' requested here
                std::__uninitialized_default_n_a(this->_M_impl._M_finish,
                     ^
/usr/bin/../lib64/gcc/x86_64-openmandriva-linux-gnu/10.1.1/../../../../include/c++/10.1.1/bits/stl_vector.h:940:4: note: in instantiation of member function 'std::vector<cv::gimpl::Op, std::allocator<cv::gimpl::Op>>::_M_default_append' requested here
          _M_default_append(__new_size - size());
          ^
../modules/gapi/src/backends/common/serialization.hpp:279:12: note: in instantiation of member function 'std::vector<cv::gimpl::Op, std::allocator<cv::gimpl::Op>>::resize' requested here
        ts.resize(sz);
           ^
../modules/gapi/src/backends/common/serialization.cpp:497:8: note: in instantiation of function template specialization 'cv::gimpl::s11n::operator>><cv::gimpl::Op>' requested here
    is >> s.m_ops >> s.m_datas >> s.m_counter >> s.m_proto;
       ^
../modules/gapi/src/compiler/gmodel.hpp:58:25: note: default constructor of 'Op' is implicitly deleted because field 'k' has a deleted default constructor
    cv::GKernel         k;
                        ^
../modules/gapi/include/opencv2/gapi/gkernel.hpp:42:23: note: default constructor of 'GKernel' is implicitly deleted because field 'inSpecs' of const-qualified type 'const cv::GSpecs' (aka 'const vector<cv::detail::ArgSpec>') would not be initialized
    const GSpecs      inSpecs;    // specs of kernel's inputs (FIXME: below)
                      ^
1 error generated.
```

--- a/modules/gapi/include/opencv2/gapi/gkernel.hpp
+++ b/modules/gapi/include/opencv2/gapi/gkernel.hpp
@@ -36,11 +36,11 @@ struct GAPI_EXPORTS GKernel
 {
     using M = std::function<GMetaArgs(const GMetaArgs &, const GArgs &)>;
 
-    const std::string name;       // kernel ID, defined by its API (signature)
-    const std::string tag;        // some (implementation-specific) tag
-    const M           outMeta;    // generic adaptor to API::outMeta(...)
-    const GShapes     outShapes;  // types (shapes) kernel's outputs
-    const GKinds      inKinds;    // kinds of kernel's inputs (fixme: below)
+    std::string name;       // kernel ID, defined by its API (signature)
+    std::string tag;        // some (implementation-specific) tag
+    M           outMeta;    // generic adaptor to API::outMeta(...)
+    GShapes     outShapes;  // types (shapes) kernel's outputs
+    GKinds      inKinds;    // kinds of kernel's inputs (fixme: below)
 };
 // TODO: It's questionable if inKinds should really be here. Instead,
 // this information could come from meta.

